docs/guides/index.html

```diff
@@ -108,8 +108,8 @@ <h4>Mailing lists</h4>
         </div>
         <div class="col-md-6">
           <h4>IRC channel (Chat)</h4>
-          <p>For occasional quick questions we have an IRC channel on <a
-            href="https://webchat.freenode.net/?channels=opencast">Freenode</a>: <em>#opencast</em>.</p>
+          <p>For occasional quick questions we have a Matrix <a
+            href="https://app.element.io/#/room/#opencast-community:matrix.org">channel</a>: <em>#opencast-community</em>.</p>
```

Comment:
This is irrelevant for docs.opencast.org and will cause merge conflicts. The landing page is now generated from 
https://github.com/opencast/opencast/blob/develop/docs/guides/.infrastructure/index.html.j2

Reply:
So, delete this whole page then?

Reply:
I wouldn't touch it in 9.x but then it's also not really important and the tests have passed so… I'll fix the merge conflict.
Merging…